### ISSUE TITLE
ChOma does not support injecting signatures into apps that are in Chinese

### ISSUE THREAD

**Issue by lucy5826** [opa334/ChOma#17]
Hello,

I have a question regarding the ChOma. It seems that ChOma does not support injecting signatures into apps that are in Chinese. Is there any plan to support Chinese apps in the future?

Thank you!

**Comment by alfiecg24** [comment]
This sounds like either an edge-case issue with Chinese characters, or not a ChOma issue at all. What is the file name of the binary you’re trying to use?

**Comment by lucy5826** [comment]
No, it's that all Chinese apps I encounter are like this, without exception. 
You can download an iOS version of the app from https://weixin.qq.com/. Alternatively, you can try downloading any app from the Chinese Apple Store.

**Comment by lucy5826** [comment]
No, it's that all Chinese apps I encounter are like this, without exception.
You can download an iOS version of the app from https://weixin.qq.com/. Alternatively, you can try downloading any app from the Chinese Apple Store.



> This sounds like either an edge-case issue with Chinese characters, or not a ChOma issue at all. What is the file name of the binary you’re trying to use?

**Comment by alfiecg24** [comment]
Downloading straight from the App Store means that the apps will be encrypted. Are you decrypting the main app before you re-sign?

**Comment by lucy5826** [comment]
> Downloading straight from the App Store means that the apps will be encrypted. Are you decrypting the main app before you re-sign?

Isn't the exploit utilized here one that allows you to sign an already installed app directly without needing to decrypt it? First, I use https://github.com/tyilo/insert_dylib to insert the dylib file into the Mach-O binary, and then copy the dylib to the app's Frameworks directory. After that, I use ct_bypass to re-sign both the binary and the dylib.
